config/default.json

```diff
@@ -41,6 +40,6 @@
     "cookieLogin": "blf-auth-state",
     "expiryInSeconds": 7200
   },
-  "hotjarId": 1036292,
+  "hotjarId": null,
```

Comment:
This is necessary otherwise `config.get` errors on an undefined key. Set to null by default and then defined for test and production environments only. We don't want to run hotjar in development.